Bottom: 9652184a0568f71e3c8d111f1ee5beca3ace4803
Top:    9e06ada3d04d8b8e361e5e036839e51c95183196
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-05-08 22:14:52 +0100

Refresh of gtk-gl-make-the-silk-layer-tra

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index a7278df..3c12833 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -108,7 +108,7 @@ ghid_set_layer (const char *name, int group, int empty)
 	    return TEST_FLAG (SHOWMASKFLAG, PCB);
 	  return 0;
 	case SL_SILK:
-	  priv->trans_lines = false;
+	  priv->trans_lines = true;
 	  if (SL_MYSIDE (idx))
 	    return PCB->ElementOn;
 	  return 0;
